.vale/styles/RedHat/Spelling.yml

```
@@ -212,9 +214,11 @@ filters:
   - Endian
   - endif
   - Esprima
+  - Etcd
```

```suggestion
  - etcd
```

for consistency with:

VALE: Use 'etcd' rather than 'Etcd'.

---

Updated.